docs/source/tutorial/running_ablation.rst

```diff
@@ -1,3 +1,502 @@
 Running an Ablation Study
--------------------------
-.. automodule:: pykeen.ablation
+=========================
+You want to find out which loss function and training approach is best-suited for your interaction model
+(model architecture)? Then performing an ablation study is the way to go!
+
+In general, an ablation study is a set of experiments in which components of a machine learning system are
+removed/replaced in order to measure the impact of these components on the system's performance. In the context of
+knowledge graph embedding models, typical ablation studies involve investigating different loss functions, training
+approaches, negative samplers, and the explicit modeling of inverse relations. For a specific model composition based on
+these components, the best set of hyper-parameter values, e.g., embedding dimension, learning rate, batch size,
+loss function-specific hyper-parameters such as the margin value in the margin ranking loss need to be determined.
+This is accomplished by a process called hyper-parameter optimization. Different approaches have been proposed, of
+which random search and grid search are very popular.
+
+
+In PyKEEN, we can define an ablation study in a configuration file (``file_name.json``) or within our own program.
+
+First, we show how to define an ablation study within a configuration file. In the following example, we define an
+ablation study for ComplEx over the Nations dataset, and name the file ``complex_nation.json``.
+In particular, we want to asses the effect of different loss functions, i.e., the binary cross entropy loss
+and the margin ranking loss, and the effect of explicitly modeling inverse relations.
+
+We start by adding metadata to our configuration file. This is helpful for a later retrospection.
+
+.. code-block:: javascript
```

Comment:
All of the examples should be from the python code interface, then at the end you can mention that there's a CLI

Reply:
Done in cec4df8e56f9c73f51bd9456c4d695c5e7d5bd02.